config/ingest.go

```diff
@@ -42,6 +53,7 @@ func NewIngest() Ingest {
 		PubSubTopic:             "/indexer/ingest/mainnet",
 		StoreBatchSize:          4096,
 		SyncTimeout:             Duration(2 * time.Hour),
+		SyncSegmentDepthLimit:   1000,
```

Comment:
did we have problems that caused us to set this so low? lets see if we can get away with 10k if we haven't